flow/core/experiment.py

```diff
@@ -1,198 +1 @@
-"""Contains an experiment class for running simulations."""
```

Comment:
not sure what happened to this file but looks unintentional

Reply:
fixed, it's a windows line ending anomaly